### ISSUE TITLE
mypy type annotations aren't always useful in docs

### ISSUE THREAD

**Issue by gchanan** [pytorch/pytorch#42120]
## 📚 Documentation

Consider the documentation for Conv2d against 1.7.0:
`
torch.nn.Conv2d(in_channels: int, out_channels: int, kernel_size: Union[T, Tuple[T, T]], stride: Union[T, Tuple[T, T]] = 1, padding: Union[T, Tuple[T, T]] = 0, dilation: Union[T, Tuple[T, T]] = 1, groups: int = 1, bias: bool = True, padding_mode: str = 'zeros')
`

The "T" type is never defined and not particularly useful for documentation, even if it is the correct mypy type.  We should have a better strategy here.

cc @jlin27

**Comment by rodrigoberriel** [comment]
This issue is no longer valid, I guess. [`torch.nn.Conv2d`](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html) docs are now (actually, since [1.8.0](https://pytorch.org/docs/1.8.0/generated/torch.nn.Conv2d.html)) rendering much _better_:

> `torch.nn.Conv2d(in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1, bias=True, padding_mode='zeros')`

@ngimel I think this one can be closed.

**Comment by ngimel** [comment]
Thanks, closing.
